### PR TITLE
dev-deploy: create neardev directory if it doesn't exist

### PR DESCRIPTION
**Pull request by mikedotexe**
If there's no `neardev` directory, `near dev-deploy` will fail with:

```
Error:  [Error: ENOENT: no such file or directory, open 'neardev/dev-account'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'neardev/dev-account'
}
error Command failed with exit code 1.
```

This is new behavior as of 0.23.0, let's address this quickly.

Tested by going into guest-book example locally:
`../near-shell/bin/near dev-deploy`

`rm -rf neardev`
then running it again
`rm neardev/dev-account*`
then running it again

### CODE REVIEW

**Comment by mikedotexe** [comment]
> I would prefer we did the equivalent of `mkdir -p` in the normal flow, rather than creating the directory in the `catch` block

That won't work for Windows, but I did consider adding the [mkdir package](https://www.npmjs.com/package/mkdirp) for this. I know there's always pushback against adding new dependencies for one small change, so I decided to do it this way.

**Comment by chadoh** [comment]
Right, I said "something like `mkdir -p` because I thought windows might give problems with actual `mkdir -p`. 

I think we could use something like this:

```js
if (!fs.existsSync('./neardev')) await fs.promises.mkdir('./neardev')
```

(Not sure I got that syntax right)

Would this work on Windows?

File: commands/dev-deploy.js [hunk 1]
```diff
@@ -63,7 +64,8 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
             }
         } catch (e) {
             if (e.code === 'ENOENT') {
-                // Ignore as it means new account needs to be created, which happens below
+                // Create neardev directory, new account will be created below
+                if (!existsSync('./neardev')) await mkdir('./neardev');
```
Comment (janedegtiareva) [inline commands/dev-deploy.js:68]: let's pull the dir name into const, so that we don't have string literal duplication

Comment (mikedotexe) [reply commands/dev-deploy.js]: pushed

**Comment by vgrichina** [comment]
This change https://github.com/near/near-shell/pull/362/commits/d4262019a84a471f5343d55e127cf331d58c0c91 only works well if all the occurrences are replaced:
https://github.com/near/near-shell/search?l=JavaScript&q=neardev

**Pull request merged**